packages/nextra-theme-docs/src/components/sidebar.tsx

```diff
@@ -163,18 +163,8 @@ function Folder({ item, anchors, onFocus, level }: FolderProps): ReactElement {
           if (clickedToggleIcon) {
             event.preventDefault()
           }
-          if (isLink) {
-            // If it's focused, we toggle it. Otherwise, always open it.
-            if (active || clickedToggleIcon) {
-              TreeState[item.route] = !open
-            } else {
-              TreeState[item.route] = true
-            }
-            rerender({})
-            return
-          }
-          if (active) return
-          TreeState[item.route] = !open
+          // If it's focused, we toggle it. Otherwise, always open it.
+          TreeState[item.route] = active || clickedToggleIcon ? !open : true
```

Comment:
this fix bug when `<button>` can't be closed 😅 (we have similar issue in v3 btw)

https://github.com/user-attachments/assets/2a0f0dd8-5979-4726-bb8e-63645ea89192